src/components-examples/material/autocomplete/autocomplete-harness/autcomplete-harness-example.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {TestBed, ComponentFixture, waitForAsync, inject} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {MatAutocompleteHarness} from '@angular/material/autocomplete/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {MatAutocompleteModule} from '@angular/material/autocomplete';
+import {AutocompleteHarnessExample} from './autocomplete-harness-example';
+import {OverlayContainer} from '@angular/cdk/overlay';
+
+describe('AutocompleteHarnessExample', () => {
+  let fixture: ComponentFixture<AutocompleteHarnessExample>;
+  let loader: HarnessLoader;
+  let overlayContainer: OverlayContainer;
+
+  beforeAll(() => {
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatAutocompleteModule],
+        declarations: [AutocompleteHarnessExample]
+      }).compileComponents();
```

Comment:
`compileComponents` is technically async, so the rest of the code needs to wait for it to complete. I'd typically do this like
```typescript
  beforeEach(waitForAsync(() => {
    TestBed.configureTestingModule({
      imports: [MatAutocompleteModule],
      declarations: [AutocompleteHarnessExample]
    }).compileComponents();  
  }));

  beforeEach(() => {
    // other setup
  });
```